izadji/src/services/blogNewsPageService.ts

```diff
@@ -8,7 +8,7 @@ const getBlogs = (
   page: number
 ) =>
   axios.get(
-    `/api/blog-pages?filters[work_program][id][$eq]=${workProgrammeId}&populate=deep&pagination[page]=${page}&pagination[pageSize]=${blogsPerPage}&filters[blog_page_tags][title][$containsi]=${searchString}`
+    `/api/blog-pages?filters[work_program][id][$eq]=${workProgrammeId}&populate=deep&pagination[page]=${page}&pagination[pageSize]=${blogsPerPage}&filters[blog_page_tags][title][$contains]=${searchString}`
```

Comment:
containsi was not a mistake https://docs.strapi.io/developer-docs/latest/developer-resources/database-apis-reference/entity-service/filter.html#containsi.
Not sure why search doesn't work but that's not it 😄